src/components/navigation/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react'
+import { View } from 'react-native'
+import { Box } from '../../structure'
+import { useStyles } from '../../../theme'
+import { Tab } from './Tab/Tab'
+
+export interface TabDescriptor {
+  /**
+   * ID of the tab.
+   */
+  id: string
```

Comment:
Will the ID have a use? If the only use is to set it to `key`, could we generate one instead to simplify the usage?

Reply:
That was my initial thought, as I found that setting an ID was too much hassle, but the thing is that if I use the tab index as a key, there could be bugs if tabs are added/removed/moved at runtime.

One way of handling it could be by having code like this:
```tsx
const realTabs = useMemo(() => {
  return tabs.map(x => ({ label: x.label, key: generateKey() }))
}, [tabs])
```

> **Note:** I would also replace the `TabDescriptor` interface with a `type TabDescriptor = string`, since there would simply be the labels on it. If we support additional options per tab later on, a `TabDescriptor` could be something like: 
`type TabDescriptor = string | { label: string; icon: IconName }`

However, what would the `generateKey` function generate? One option could be to have a UUID generated. Either through an external package (maybe like `uuid`, which I doubt work in RN (since there is no `crypto` support), as there are UUID packages specifically for React Native). We could duplicate `uuid` package using [`expo-crypto`](https://docs.expo.io/versions/latest/sdk/crypto/).

We could also use this function which I found on StackOverflow: 
```
function guidGenerator() {
    var S4 = function() {
       return (((1+Math.random())*0x10000)|0).toString(16).substring(1);
    };
    return (S4()+S4()+"-"+S4()+"-"+S4()+"-"+S4()+"-"+S4()+S4()+S4());
}
```

I think it would be simple and straightforward enough for our use case, even if not perfect.

Reply:
Changed behavior to use Tab text as `key`. Tabs text should be unique anyway, so requiring an ID seems unnecessary.